packages/fds-components/src/style/buttons/buttonGroup.css

```diff
@@ -0,0 +1,70 @@
+:root {
+  --button-group-timeout: 250ms;
+}
+
+.btn-group {
```

Comment:
*Not a blocker*

I've been using hyphens as a convention to denote a `parent-child` relationship. We can always go back and change these later after we discuss what kind of naming conventions we want to use going forward

Reply:
If you have a pre-existing convention, I'd rather follow it 🙂 are you ok with .btngroup and .groupbtn?